src/.vuepress/components/DownloadButtons.vue

```diff
@@ -42,7 +42,7 @@ export default {
 		}
 		try {
 			const { data } = await this.$store.dispatch("getPreviewReleaseData");
-			const apkAsset = data.assets.find((a) => a.name.includes(".apk"));
+			const apkAsset = data.assets.find((a) => /^tachiyomi-r\d{4}.apk/.test(a.name));
```

Comment:
This will break the day we hit 10000 commits (although that's pretty far away). It should be safe to just rely on the `tachiyomi-r` prefix.

Reply:
> This will break the day we hit 10000 commits (although that's pretty far away). It should be safe to just rely on the `tachiyomi-r` prefix.

That should be fine.
```suggestion
			const apkAsset = data.assets.find((a) => /^tachiyomi-r\d{4,5}.apk/.test(a.name));
```